triton-vm/src/table/processor_table.rs

```diff
@@ -1342,14 +1328,11 @@ impl RowPairConstraints {
             (self.one() - self.hv0()) * (self.ramv_next() - self.ramv());
 
         vec![
+            vec![hv0_is_0_or_1, st10_is_shifted_1_bit_right],
+            maybe_copied,
             vec![
-                hv0_is_0_or_1,
-                st12_decomposes_to_hvs,
-                st12_becomes_shifted_1_bit_right,
-            ],
-            left_siblings_remain_left,
-            right_siblings_are_copied_right,
-            vec![
+                st11_does_not_change,
+                st12_does_not_change,
                 st13_does_not_change,
```

Comment:
I feel that with as telling variable names as we have, we probably don't need this `vec![vec![…]].concat()`. Maybe a line break to separate conceptual groups, but that should suffice.

Reply:
Did not at all mean to throw shade. :relieved: